packages/ui/src/views/GameSettings.vue

```diff
@@ -11,6 +11,7 @@
           v-model="username"
         />
         <SectionHeader title="Instructions" />
+        <!-- TODO: Update copies -->
```

Comment:
1. JOIN THE ETHDENVER
OFFICIAL GAME
Visit the Witnet + WittyPixels booth at the EthDenver registration area to receive your own limited edition player palette pendant. Then immediately open wittypixels.art in your phone and scan the QR code printed on your pendant. The first to scan it owns it!

2. GET YOUR PALETTE
SCANNED BY OTHERS
Earn pixels of distinct colors by getting the QR code on your palette scanned by players with different palette colors. Here are some rules:
 - There are 6 different palette colors:
 (colors)
 - Scanning someone for the 1st time gives 10px, 2nd time is 5px, then 3px, 2px, and 1px. Make sure to scan new players all the time!
 - You can only scan or be scanned every 2 minutes.
 - The same 2 players can only repeat scanning every 30 minutes.

3. PLACE YOUR PIXELS
INTO THE CANVAS
Place pixels of the different colors into the shared virtual canvas. When selecting a color, you can also pick lighter and darker shades for more creative control. Bring out the artist in you, and try to coordinate with other players to paint a masterpiece together!

4. SEEK THE BONUSES
Try to spot the bonus QR codes located across the venue and at sponsors booths. Every time you scan a new bonus, any further scan is worth 2x the pixels for 15 minutes!

5. TURN YOUR PIXELS
INTO NFT FRACTIONS
On March 5, a snapshot of the canvas will be minted as a single ERC-721 NFT, and fractions of it will be distributed to the players as an ERC-20 named $WPX. The more pixels you get into the final artwork, the more $WPX fractions you will own!

6. WITNESS A HISTORIC
NFT AUCTION
Right after minting, the fractionalized NFT will be sold at a public Dutch auction, where one and only "patron" will be able to purchase the canvas NFT and own this one-of-a-kind piece of crypto history.

7. SWAP YOUR $WPX
FOR REAL $ETH!
50% of the $ETH from the auction will be donated to TheGivingBlock's Ukraine Emergency Response Fund.
The other 50% will be distributed to the players proportionally to the $WPX fractions they hold. That is, on top of the fun and the networking, you may go back home with some extra $ETH in your wallet!

Reply:
OFFICIAL GAME
Join the official game of EthDenver by visiting the Witnet + WittyPixels booth at the registration area. Receive a limited edition player palette pendant and scan the QR code on it with wittypixels.art on your phone. The first person to scan it will own it!

GET YOUR PALETTE
Get pixels of distinct colors by having the QR code on your palette scanned by players with different palette colors. There are six different palette colors: (Orange, Yellow, Green, Blue, Red, and Purple). You'll get 10 pixels for the first scan, 5 for the second, and then 3, 2, and 1 for subsequent scans. Remember to scan new players all the time! Note that you can only scan or be scanned every two minutes, and the same two players can only repeat scanning every 30 minutes.

PLACE YOUR PIXELS
Place pixels of different colors into the shared virtual canvas to create a masterpiece together with other players. You can pick lighter and darker shades for more creative control.

SEEK THE BONUSES
[no change]

TURN YOUR PIXELS INTO NFT FRACTIONS
[no change]

WITNESS A HISTORIC NFT AUCTION
After minting, the fractionalized NFT will be sold at a public Dutch auction, where one "patron" will be able to purchase the canvas NFT and own this one-of-a-kind piece of crypto history. 50% of the proceeds from the auction will be donated to TheGivingBlock's Ukraine Emergency Response Fund, while the other 50% will be distributed to players proportionally to the $WPX fractions they hold. So not only will you have fun and network with others, but you may also go home with extra $ETH in your wallet!